src/catalog/src/expr_cache.rs

```diff
@@ -77,55 +75,27 @@ struct ExpressionCodec;
 
 impl DurableCacheCodec for ExpressionCodec {
     type Key = CacheKey;
-    // We use a raw JSON string instead of `Expressions` so that there is no backwards compatibility
+    // We use a raw bytes instead of `Expressions` so that there is no backwards compatibility
     // requirement on `Expressions` between versions.
-    type Val = String;
-    type KeyCodec = SourceData;
-    type ValCodec = ();
+    type Val = Vec<u8>;
+    type KeyCodec = Vec<u8>;
+    type ValCodec = Vec<u8>;
```

Comment:
Is the expression cache being used in Prod or Staging? yet? If so, I don't think changing the schemas like this is safe, unless you tear down the environment

Reply:
It is not used in prod or staging